ssl/t1_lib.c

```diff
@@ -705,7 +705,7 @@ static int gid_cb(const char *elem, int len, void *arg)
     gid_cb_st *garg = arg;
     size_t i;
     uint16_t gid = 0;
-    char etmp[20];
+    char etmp[64];
```

Comment:
Should the constant be a # define somewhere in some include file at least?

Reply:
There are constants like this everywhere in the code.  If it were to be a `#define`  , I would imagine that to be part of a wider code cleanup effort.  

However, I am willing to do it.  Can you suggest a name and location?  I'm thinking `ssl_local.h`.

Reply:
I defined it right after `MAX_GROUPLIST`